graph/encoding/dot/decode.go

```diff
@@ -486,10 +509,23 @@ func addEdgeAttrs(edge graph.Edge, attrs []*ast.Attr) {
 	for _, attr := range attrs {
 		a := encoding.Attribute{
 			Key:   attr.Key,
-			Value: attr.Val,
+			Value: unquoteAttr(attr.Val),
 		}
 		if err := e.SetAttribute(a); err != nil {
 			panic(fmt.Errorf("unable to unmarshal edge DOT attribute (%s=%s): %v", a.Key, a.Value, err))
 		}
 	}
 }
+
+// unquoteAttr unquotes the given string if needed in the context of an
+// attribute value. If s is not already quoted the original string is returned.
+func unquoteAttr(s string) string {
+	// Unquote quoted string if possible.
+	if t, err := strconv.Unquote(s); err == nil {
```

Comment:
We need to check that the string is quoted with `"` since `strconv.Unquote` will also unquote backtick quotes (which I can't do inline AFAICS), but are invalid for DOT.

Reply:
Ah, you are right. Added this check.